packages/gestalt/src/Tooltip/InternalTooltip.js

```diff
@@ -102,6 +102,26 @@ export default function InternalTooltip({
     dispatch({ type: 'hoverOutText' });
   }, mouseLeaveDelay);
 
+  const getTooltipText = () => {
+    if (Array.isArray(text)) {
+      const lines = [];
+      // first and last line should not have a <p> tag, (adds extra padding)
+      text.map((line, idx) =>
+        lines.push(
```

Comment:
yo don't really need the push into line right?

you can just return and use the map  const a = text.map()

Reply:
@rlingineni maybe we should also check the type of `line`? When someone disables flow type checking and passes a JSX or smth, should it be our concern?